first_timers/first_timers.py

```diff
@@ -38,6 +38,21 @@ def get_first_timer_issues():
     return items
 
 
+def add_repo_languages(issues):
+    """Adds the repo languages to the issues list."""
+    for issue in issues:
+        query_languages = issue['repository_url'] + '/languages'
+        res = requests.get(query_languages)
+        if res.status_code == 403:
+            warnings.warn('Rate limit reached getting languages')
+            return issues
+        elif res.ok:
+            issue['languages'] = res.json()
+        else:
+            raise RuntimeError('Could not handle response: ' + str(res) + ' from the API.')
```

Comment:
I am not very sure about raising the exception.
I feel just logging it would be fine. As raising it might break while tweeting and we do not want tweeting to be dependent on this.

Any thoughts?

Reply:
Ok. I'll change it to a warning to not break the tweeting.